### ISSUE TITLE
CHANGE_LOG missing: BE_ListFilesInFolder is not in version 4.2.0, was it renamed to BE_FileListFolder ?

### ISSUE THREAD

**Issue by wujoho** [GoyaPtyLtd/BaseElements-Plugin#202]
If _BE_ListFilesInFolder_ was renamed to _BE_FileListFolder_, the new function is not compatible with _BE_ListFilesInFolder_'s parameters?

**Comment by nickorr** [comment]
A rename doesn't affect the parameters.  Not sure what you're referring to here...

Also if you need plugin support please use : https://docs.baseelementsplugin.com  There's a contact link on that page.  

We try to keep GitHub issues to things related to the c++ code, or compiling etc.

Cheers,
Nick

**Comment by wujoho** [comment]
sorry for the confusion, hope the below makes it a bit clearer here.

_BE_ListFilesInFolder_ was in old version like 3.1 but not in latest version, and I couldn't find its changing history like when removed or rename in the CHANGE_LOG [page](https://github.com/GoyaPtyLtd/BaseElements-Plugin/blob/master/CHANGE_LOG)

Also if _BE_ListFilesInFolder_ was renamed to _BE_FileListFolder_, then they take different parameters, it was valid to use _BE_ListFilesInFolder ( folderPath ; "*.txt" )_ in old version, but in new version _BE_FileListFolder_ doesn't take "*.txt" for the second parameter.

**Comment by nickorr** [comment]
I don't believe that it ever had an option for limiting the file names returned.  We've had options to do that generally via regex, but never in this function, that I'm aware of.

**Comment by wujoho** [comment]
yes, you are correct, i double checked this function in version 3.1.0, it doesn't support filtering by file type like the above, it was a misuse in our code, sorry for the confusion.

So it's just the below part:
_BE_ListFilesInFolder_ was found in old version like 3.1, but couldn't find it in the latest version, also couldn't find its changing history like when it was removed or was rename in the [CHANGE_LOG](https://github.com/GoyaPtyLtd/BaseElements-Plugin/blob/master/CHANGE_LOG)
